migrations/1593194295169_add-status-to-action-items.ts

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate';
+
+export const shorthands: ColumnDefinitions | undefined = undefined;
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+    pgm.addColumn('action_items', {
+        status: { type: 'text' }
+    });
+}
```

Comment:
Should this be nullable?
How will we handle the existing action items?

If we query with this `SELECT * FROM action_items WHERE workspace_id = $1 AND channel_id = $2 AND status='OPEN'`, we will effectively ignore the existing action items

Reply:
just a simple `update action_items set status="OPEN"` should do the trick

Reply:
Could we make the column non-nullable then? And set the default value to `"OPEN"`?
IIRC, that will _atomically_ add the column and set the value